src/metabase/api/action.clj

```diff
@@ -93,9 +93,10 @@
 
 (api/defendpoint PUT "/:id"
   [id :as
-   {{:keys [type name description model_id parameters parameter_mappings visualization_settings
-            kind database_id dataset_query template response_handle error_handle] :as action} :body}]
+   {{:keys [archived database_id dataset_query description error_handle kind model_id name parameter_mappings parameters
+            response_handle template type visualization_settings] :as action} :body}]
```

Comment:
maybe this is dumb, but I got tired of comparing two lists of params to see what was different between them, so I'm on a little kick for alphabetizing it.